components/camera/videosource/webcam.go

```diff
@@ -54,15 +59,22 @@ func init() {
 	}
 }
 
-func getVideoDrivers() []driver.Driver {
-	return driver.GetManager().Query(driver.FilterVideoRecorder())
-}
-
-// CameraConfig is collection of configuration options for a camera.
-type CameraConfig struct {
```

Comment:
I don't know what this is. It's not used anywhere and it's confusing to keep it here

Reply:
Does discovery still work without this? https://github.com/viamrobotics/api/blob/cdaf1d58f503c0c373a3a3e3d58d8fe2fc83763a/proto/viam/component/camera/v1/camera.proto#L149

It's the proto message the front end uses to nicely render the options returned from webcams' discover function.

Reply:
hmm yeah I think it does since I was able to find all the webcams for me to select in the UI in my manual test I outlined

Reply:
Okay, app must be using a wrapper for this message in typescript or the raw proto. Thanks for checking.

Reply:
Nice! That struct has been around since 2022-06-01 -- probably became unused at some point.

I guess the linter does not flag unused since it public...